assets/sass/components/overlay-notification/_googlesitekit-overlay-notification.scss

```diff
@@ -98,4 +98,18 @@
 			padding-top: 0;
 		}
 	}
+
+	&.googlesitekit--overlay-notification--reader-revenue-manager {
+		max-width: 348px;
+
+		.mdc-button--raised {
+			@extend .mdc-button--raised;
```

Comment:
This looks redundant. Do you think this is necessary?

Reply:
Removed this one.